lib/ansible/modules/cloud/docker/docker_swarm.py

```diff
@@ -391,20 +395,16 @@ def __call__(self):
 
         choice_map.get(self.state)()
 
+        if self.state == 'inspect':
```

Comment:
Can you move this two lines up, so that the dispatching comes after the deprecation warning?

Reply:
No problem :)

Reply:
Thanks! I think it's better to issue the warning before "real action" starts (so the errors don't block the warning).